rllib/agents/dqn/simple_q_tf_policy.py

```diff
@@ -55,15 +55,15 @@ def build_q_models(policy, obs_space, action_space, config):
         action_space=action_space,
         num_outputs=action_space.n,
         model_config=config["model"],
-        framework="torch" if config["use_pytorch"] else "tf",
+        framework="torch" if config["framework"] == "torch" else "tf",
```

Comment:
Can these be `framework=config["framework"]`?

Reply:
Good idea. Then we have to change the ModelCatalog to be more flexible wrt tf/tfe.

Reply:
done